### PR TITLE
blog css cleanup

### PR DESCRIPTION
**Pull request by flynnduism**
This PR updates the website css to improve on the appearance of the blog pages. Also links to the blog RSS feed.

Screen Shot 2021-05-17 at 1 29 25 PM

### CODE REVIEW

File: docs/themes/porter/assets/sass/helm-blog.scss [hunk 1]
```diff
@@ -60,6 +7,7 @@
       .content-wrap {
         h1 {
           margin-top: 0;
+          color: $gray;
```
Comment (carolynvs) [inline docs/themes/porter/assets/sass/helm-blog.scss:10]: I can't tell if this change was intentional? I'm seeing the listing page for the blog give each blog title entry a background color which looks a bit off to me.

Screen Shot 2021-05-17 at 3 50 41 PM

Comment (flynnduism) [reply docs/themes/porter/assets/sass/helm-blog.scss:10]: I've reverted these color changes back. I'd like to introduce more of the Porter colors from the branding last year, but that's better addressed separately I think to tweak the theme overall. I'll stick to the bug fixes for now

Screen Shot 2021-05-17 at 1 56 17 PM

Comment (carolynvs) [reply docs/themes/porter/assets/sass/helm-blog.scss:10]: I love the purple, noticed that immediately on the dates. So looking forward to the next PR. 👍

**Review by carolynvs — APPROVED** [review]
Looks so much better, thank you!

**Pull request merged**